kubetest/e2e.go

```diff
@@ -241,9 +241,6 @@ func run(deploy deployer, o options) error {
 
 func listNodes(dump string) error {
 	b, err := output(exec.Command("./cluster/kubectl.sh", "--match-server-version=false", "get", "nodes", "-oyaml"))
-	if verbose {
```

Comment:
I was assuming that the program is likely to write something to stdout even if it returns a non-zero exit code. 

Not a big deal. We can add this back if it causes problems